style.css

```diff
@@ -0,0 +1,78 @@
+@import url("https://fonts.googleapis.com/css2?family=IBM+Plex+Sans+KR:wght@400;500;700&display=swap");
```

Comment:
Don't use `@import `in your stylesheet use its alternative `<link>` in HTML file `<head>`
It slows down your page and reduce accessibility and affects SEO